### PR TITLE
Simplify DisplayGroupView on master process

### PR DESCRIPTION
**Pull request by rdumusc**
This decouples the rendering of the background wall displays (fixed at startup time, read from configuration) from the DisplayGroup.

### CODE REVIEW

File: apps/DisplayCluster/DisplayGroupView.cpp [hunk 1]
```diff
@@ -52,26 +53,38 @@
...
-const QString MASTER_WINDOW_OBJECT_NAME( "MasterContentWindow" );
+const QString WALL_OBJECT_NAME( "Wall" );
 const QUrl QML_CONTENTWINDOW_URL( "qrc:/qml/master/MasterContentWindow.qml" );
 const QUrl QML_DISPLAYGROUP_URL( "qrc:/qml/master/MasterDisplayGroup.qml" );
 const QUrl QML_BACKGROUND_URL( "qrc:/qml/master/DisplayGroupBackground.qml" );
 }
 
-DisplayGroupView::DisplayGroupView( OptionsPtr options )
+DisplayGroupView::DisplayGroupView( OptionsPtr options,
+                                    MasterConfiguration& config )
```
Comment (tribal-tec) [inline apps/DisplayCluster/DisplayGroupView.cpp]: const& ?

Comment (chevtche) [inline apps/DisplayCluster/DisplayGroupView.cpp]: const

File: apps/DisplayCluster/DisplayGroupView.h [hunk 2]
```diff
@@ -59,14 +59,13 @@ class DisplayGroupView : public QQuickView
 
 public:
     /** Constructor. */
-    DisplayGroupView( OptionsPtr options );
+    DisplayGroupView( OptionsPtr options, MasterConfiguration& config );
```
Comment (chevtche) [inline apps/DisplayCluster/DisplayGroupView.h]: const

File: apps/DisplayCluster/DisplayGroupView.cpp (hunk 1 above)
Comment (rdumusc) [inline apps/DisplayCluster/DisplayGroupView.cpp]: done

**Pull request merged**